### ISSUE TITLE
Mangle (Cat Form) buff not working.

### ISSUE THREAD

**Issue by guischmo** [dalaranwow/dalaran-wow#4948]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**:  The 30% Buff on bleeds provided by Mangle (Cat) does not work

**Expected behaviour**: When applying Rake or Rip, if the mob has the mangle debuff on him, they should tick for 30% more than without the buff.

Two other druids I whisped confirmed This. This is a pretty big issue for feral druids because 30% dps lost on our Dots which are the main source of damage for us is quite important.

**Steps to reproduce the problem**:

1. Got to Dummy and Rake him, check the dot damage.
2. Apply Mangle and Rake him again, check the dot damage, there is no difference (beware of trinkets procs and stuff). Be sure to Mangle first then rake.
3. Try it with Rip too

![2](https://user-images.githubusercontent.com/28254131/31933757-5732ae12-b8aa-11e7-90a2-305c024ea7ae.png)
![1](https://user-images.githubusercontent.com/28254131/31933758-5762f888-b8aa-11e7-940b-1454bba31575.png)

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Fiky995** [comment]
I've tested it too, even mangle(bear) doesn't work
Rake with no mangle
![wowscrnshot_102417_120117](https://user-images.githubusercontent.com/13967525/31936981-76e456c6-b8b3-11e7-8b37-5334e77c61bb.jpg)
Rake with mangle(cat)
![wowscrnshot_102417_120142](https://user-images.githubusercontent.com/13967525/31937019-90089d10-b8b3-11e7-955f-3fa952ac2ffa.jpg)
Rake with mangle(Bear)
![wowscrnshot_102417_120200](https://user-images.githubusercontent.com/13967525/31937036-9a29d700-b8b3-11e7-9919-015f718419e2.jpg)
Always 1009 dmg

**Comment by Lothloryen** [comment]
Hello there, when did start happening? Thank you very much.

**Comment by guischmo** [comment]
Hi, I have no idea. I noticed the bug this morning and never thought about lookig at the Mangle effect before that day.

**Comment by Lothloryen** [comment]
Hello again, @robinsch found what was causing it - it has been already corrected. Thank you very much for your collaboration.
